### DISCUSSION TITLE
Language files, missing/hidden key-value pairs?

### DISCUSSION THREAD

**Discussion by kzsa** [directus/directus#7793] [Archive]
Hi,
Not a big deal, but the "Administrator" and Description > "Initial administrative role with unrestricted App/API access" under 192.168.100.1:8055/admin/settings/roles/ are not parts of the translation list.
Br.: kzsa

![missing](https://user-images.githubusercontent.com/59151228/131961387-410f48e6-c2cc-4064-b6a8-702000b7a40b.jpg)
![cr](https://user-images.githubusercontent.com/59151228/131961400-35d2fdb4-1a3a-432f-b23e-c0feb2bb8cd9.jpg)

**Comment by rijkvanzanten** [comment]
@jaycammarano Wanna see where that string is coming from?

**Comment by kzsa** [comment]
![directus – 2021-09-03 18-30-14](https://user-images.githubusercontent.com/59151228/132038788-424a3c5c-9531-40cf-9823-53f2fe87a01d.png)

:D

**Reply by kzsa** [reply]
Hardcoded. 🤷‍♂️ But at least it cannot be mistranslated! 😂

**Reply by rijkvanzanten** [reply]
Right that makes sense. It's technically "user generated" content, though we insert the first role so just enter a good default. We've been working more and more with translatable-user-values, so this would be one more of those 🙂 (cc @benhaynes)

**Reply by benhaynes** [reply]
Yeah, I think this is fine for now since the user/admin can update as needed. Eventually, we'll have the translation strings configurable in the App, and this will be properly resolved.
